src/aavrutiDJun20/array/IntersectionOfTwoArray.java

```diff
@@ -0,0 +1,74 @@
+package aavrutiDJun20.array;
+
+public class IntersectionOfTwoArray {
+	int duplicateNumberCnt = 0;
+	
+	//Store count of all the duplicate numbers from two arrays and store it in temporary array
+	double[] findIntersectionOfArray(double[] array1,double[] array2) {
+		int numberCnt = 0;
+		int arrayLength = 0;
+		
+		if(array1.length>=array2.length)
+			arrayLength = array1.length;
+		else
+			arrayLength = array2.length;
+		
+		double[] tempIntersectionArray = new double[arrayLength];
+		String[] str = new String[arrayLength];
+		str[0] = "";
+		
+		for(int outerIndex=0;outerIndex<array1.length;outerIndex++) {			
+				for(int innerIndex=0;innerIndex<array2.length-1;innerIndex++) {
+					if(array1[outerIndex] == array2[innerIndex]) {
+						tempIntersectionArray[numberCnt] = array2[innerIndex];
+						numberCnt++;
+					}
+				}
+		}
+		
+		for(int outerIndex=0;outerIndex<numberCnt;outerIndex++) {
```

Comment:
Replacement of line number 29 to 44. I removed String array and use double array. 

double[] str = new double[arrayLength];
		for (int outerIndex = 0; outerIndex < numberCnt; outerIndex++) {
			boolean isNumberDuplicate = true;
			for (int innerIndex = 0; innerIndex < duplicateNumberCnt; innerIndex++) {
				if (tempIntersectionArray[outerIndex] == str[innerIndex]) {
					isNumberDuplicate = false;
					break;
				}
			}
			if (isNumberDuplicate == true) {
				str[duplicateNumberCnt] = tempIntersectionArray[outerIndex];
				duplicateNumberCnt++;
			}

		}
		return str;